pkg/kubelet/kubelet.go

```diff
@@ -2731,7 +2741,7 @@ func (kl *Kubelet) setNodeStatus(node *api.Node) error {
 			kl.recordNodeStatusEvent("NodeOutOfDisk")
 		}
 	} else {
-		if nodeOODCondition.Status != api.ConditionFalse {
+		if nodeOODCondition.Status != api.ConditionFalse && kl.clock.Since(nodeOODCondition.LastTransitionTime.Time) >= kl.outOfDiskTransitionFrequency {
```

Comment:
maybe log a message if
nodeOODCondition.Status != api.ConditionFalse
but the clock condition is not true yet?